showcase.json

```diff
@@ -3245,5 +3245,17 @@
     ],
     "usesInternetIdentity": true,
     "github": "https://github.com/id-daniel-mccoy/asset-app"
-  }
-]
\ No newline at end of file
+  },{
+  "name": "user controlled mind and body canister”,
+  "website": "https://github.com/blueberryxtech/ic-mind-body-canister”,
+  "oneLiner": "tooling for uploading and accessing mind and body data from a firebase API onto ICP canister, blueberry data as a reference example",
+  "tags": [
+      "Tools / Infrastructure"
+    ],
+  "description": "POC of a decentralized mind and body data management tooling that leverages Internet Computer (IC) technology. This POC provides methods and interface to submit HTTP Outcall requests to a proxy API that interfaces with a Firebase FIrestore database. It provides an example front end react application that interacts with a motoko backend.”,
+  "stats": "100% on IC mainnet”,
```

Comment:
Can you please remove this as a stat? We normally suggest a stat to list the number of users, etc.